playwright_test/utils/kmLocators.js

```diff
@@ -0,0 +1,22 @@
+
+/*
+ * kmLocators.js
+ * playwright_test/utils
+ * Created by Archit on 21/12/2011. 
+ */
+
+export const locators = {
+    envBtn : '//label[@for="test"]',
+    appIdField : '//*[@id="root"]/div/div/div[2]/div/div/input',
```

Comment:
@archit2602  please check the XPath, do not use this much div. Could you try to move with the nearest attribute?

Reply:
fixed